### ISSUE TITLE
CSS import is breaking the build on Next.js

### ISSUE THREAD

**Issue by urkopineda** [lumada-design/hv-uikit-react#3715]
### Which version of NEXT UI Kit are you opening this bug for?

v5.x

### Latest version

- [X] I have tested the latest version

### No Hitachi confidential content

- [X] I'm not including any Hitachi confidential content

### Current behavior 😯

Since `@hitachivantara/uikit-react-lab@5.5.0`, importing `HvStepNavigation` is breaking the build on Next.js. The error shown when building the project seems to be related to a CSS import. The last working version with Next.js is `@hitachivantara/uikit-react-lab@5.4.50`.

```
./node_modules/reactflow/dist/style.css
Global CSS cannot be imported from within node_modules.
Read more: https://nextjs.org/docs/messages/css-npm
Location: node_modules/@hitachivantara/uikit-react-lab/dist/esm/components/Flow/DroppableFlow.js

Import trace for requested module:
./node_modules/reactflow/dist/style.css
./node_modules/@hitachivantara/uikit-react-lab/dist/esm/components/Flow/DroppableFlow.js
./node_modules/@hitachivantara/uikit-react-lab/dist/esm/components/Flow/Flow.js
./node_modules/@hitachivantara/uikit-react-lab/dist/esm/index.js
./src/components/shared/Wizard/Wizard.tsx

./node_modules/reactflow/dist/style.css
Module build failed: Error: Final loader (./node_modules/next/dist/build/webpack/loaders/error-loader.js) didn't return a Buffer or String
    at processResult (/Users/upinedaolmo/Projects/file-administrator/ch2-nas-manager-ui/node_modules/next/dist/compiled/webpack/bundle5.js:28:398847)
    at /Users/upinedaolmo/Projects/file-administrator/ch2-nas-manager-ui/node_modules/next/dist/compiled/webpack/bundle5.js:28:400317
    at /Users/upinedaolmo/Projects/file-administrator/ch2-nas-manager-ui/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:8727
    at iterateNormalLoaders (/Users/upinedaolmo/Projects/file-administrator/ch2-nas-manager-ui/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:5565)
    at iterateNormalLoaders (/Users/upinedaolmo/Projects/file-administrator/ch2-nas-manager-ui/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:5650)
    at /Users/upinedaolmo/Projects/file-administrator/ch2-nas-manager-ui/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:5879
    at runSyncOrAsync (/Users/upinedaolmo/Projects/file-administrator/ch2-nas-manager-ui/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:4185)
    at iterateNormalLoaders (/Users/upinedaolmo/Projects/file-administrator/ch2-nas-manager-ui/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:5782)
    at /Users/upinedaolmo/Projects/file-administrator/ch2-nas-manager-ui/node_modules/next/dist/compiled/loader-runner/LoaderRunner.js:1:5426
    at /Users/upinedaolmo/Projects/file-administrator/ch2-nas-manager-ui/node_modules/next/dist/compiled/webpack/bundle5.js:28:399792

Import trace for requested module:
./node_modules/reactflow/dist/style.css
./node_modules/@hitachivantara/uikit-react-lab/dist/esm/components/Flow/DroppableFlow.js
./node_modules/@hitachivantara/uikit-react-lab/dist/esm/components/Flow/Flow.js
./node_modules/@hitachivantara/uikit-react-lab/dist/esm/index.js
./src/components/shared/Wizard/Wizard.tsx


> Build failed because of webpack errors
```

### Expected behavior 🤔

The project should build successfully.

### Steps to reproduce 🕹

Sandbox link: https://codesandbox.io/p/sandbox/zealous-meadow-r7hc8y

**Comment by plagoa** [comment]
Thanks @urkopineda , looking into it.

**Comment by MEsteves22** [comment]
Hello @urkopineda, a fix for this is available on [@hitachivantara/uikit-react-lab@5.11.0](https://github.com/lumada-design/hv-uikit-react/releases/tag/%40hitachivantara%2Fuikit-react-lab%405.11.0). Thanks for reporting this issue 🙏
